lib/errors.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const errorTitle = (code) => `${code}_ERROR`;
+const errorMsg = (code) => `There is a ${code}_ERROR`;
```

Comment:
I'd actually argue for moving the braces around the single argument
```
const errorMsg = code => `There is a ${code}_ERROR`;
```

However, I won't mention dropping the semi colon just yet...